File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.java
import java.util.*;

class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        Arrays.sort(score);
        int applesAdded = 0;
        
        int startIndex = score.length % m;
        for (int i = startIndex; i < score.length; i += m) {
            answer += score[i] * m;
        }
    
        return answer;
    }
}

File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    answer = 0
    score = sorted(score)
    boxes = score[len(score) % m::m]
    min_value = score[len(score) % m::m]
    for min in min_value:
        answer += min * m
    
 
    return answer